pandas/tests/test_tseries.py

```diff
@@ -423,6 +423,14 @@ def test_series_bin_grouper():
     exp_counts = np.array([3, 3, 4], dtype=np.int64)
     assert_almost_equal(counts, exp_counts)
 
+def test_resample_with_timedeltaindex():
```

Comment:
should be in ``pandas/tseries/tests/test_resample``. no need to import things, if they are not in the namespace import at the top of the file (but they are already in any event)